draft-jhoyla-req-mtls-flag.md

```diff
@@ -144,6 +144,54 @@ terminate the connection with a fatal `illegal_parameter` alert.
 
 # Security Considerations
 
+## Authentication Properties
+
+TLS client authentication is bound to a particular TLS session. This means that
+a malicious server cannot manipulate a client in such a way that they can
+impersonate said client to any other server. This is due to the matching
+sessions property of TLS {{?DFGS21=DOI.10.1007/s00145-021-09384-1}}.
+
+This differs sharply from e.g. HTTP Signatures
+{{?I-D.draft-meunier-web-bot-auth-architecture}}, where the receiving server
+can relay a signature header to another server. In HTTP Signatures this is
+intentional, as it enables the use of proxies.
+
+This is because they authenticate different things. TLS client authentication
+identifies the actor you are talking to directly, whereas HTTP signatures
+identifies the actor that triggered some request, but not necessarily the actor
+you are in direct contact with.
+
+We describe the difference between these two properties as "principal
+authentication" and "agent authentication". Principals are the initiators of a
+particular sequence of actions. It is expected for their authentication
+information to be proxied from agent to agent as the relevant actions play out.
+For example a crawler might request a webpage from a CDN, and the CDN may relay
+the crawler's authentication, in the form of a `Signature` header, to the
+origin server. Agents, on the other hand, are the immediate participants in a
+protocol. Each agent in a chain authenticates itself only to its direct peers,
+i.e. those with whom it establishes a TLS connection.
```

Comment:
i really like this section an framing
if you're willing, it would make for a great addition to the glossary and the related consideration discussed https://github.com/thibmeu/draft-meunier-glossary-somehow/issues/5